check_test.go

```diff
@@ -151,6 +151,34 @@ func TestCheck(t *testing.T) {
 				resource.NewVersion(testPullRequests[6]),
 			},
 		},
+		{
+			description: "check supports specifying base context",
```

Comment:
I don't think we need these tests since it's covered in `put` (and `check` does not use them).